SPECS/python3/python3.spec

```
@@ -2,12 +2,12 @@
 %global __brp_python_bytecompile %{nil}
 %global majmin 3.9
```

Since we're here:
```suggestion
%define majmin %(echo %{version} | cut -d. -f1-2)
```

Not a blocker or anything, though.

---

Ah, fun fact, I tried this before and it broke the build. So, I'll go ahead and stick with it as-is for now!